jenkins/Jenkinsfile-release

```diff
@@ -1,11 +1,14 @@
 library('private-pipeline-library')
 library('jenkins-shared')
 
-mavenReleasePipeline(
+def settings =[
+    deployBranch: 'main',
```

Comment:
I don't believe deployBranch is an available option on the mavenReleasePipeline function; only mavenSnapshotPipeline.

Reply:
Oh, got it 👍